### ISSUE TITLE
npm run webpack error on windows8

### ISSUE THREAD

**Issue by cihadturhan** [Widen/fullstack-react#1]
When I execute `npm run webpack`, it throws following error.
Do you have any idea why does it happen?

```
C:\Users\cihad\Desktop\fullstack-react>npm run webpack

> fullstack-react@0.0.1 webpack C:\Users\cihad\Desktop\fullstack-react
> $(npm bin)/webpack -w

'$' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! fullstack-react@0.0.1 webpack: `$(npm bin)/webpack -w`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the fullstack-react@0.0.1 webpack script.
npm ERR! This is most likely a problem with the fullstack-react package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     $(npm bin)/webpack -w
npm ERR! You can get their info via:
npm ERR!     npm owner ls fullstack-react
npm ERR! There is likely additional logging output above.
npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "webpack"
npm ERR! cwd C:\Users\cihad\Desktop\fullstack-react
npm ERR! node -v v0.10.34
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Users\cihad\Desktop\fullstack-react\npm-debug.log
npm ERR! not ok code 0

```

**Comment by rnicholus** [comment]
That variable expansion only works in *nix environments. I haven't used Windows in quite some time, so I can't remember offhand how to do this, but `npm bin` will resolve to the binary subdirectory of your local node_modules directory. So, with this in mind, you can either use whatever syntax windows requires to expand that output as part of your command to run webpack, or you can simply run `webpack` inside the binary directory of node_modules.

**Comment by cihadturhan** [comment]
I installed modules globally with `-g` option and simply changed `$(npm bin)/webpack -w` to `webpack -w` that resolved the issue.

**Comment by rnicholus** [comment]
>I installed modules globally with -g option 

I would highly suggest you _not_ do this. This is generally considered bad practice and will definitely cause you problems down the line once you have multiple npm-based projects with varying dependencies.

**Comment by rnicholus** [comment]
Note that this was fixed in #2. I also updated to blog post with cross-platform instructions for running webpack.
